### ISSUE TITLE
audio crash during third session (2019-11-26)

### ISSUE THREAD

**Issue by yacineMahdid** [BIAPT/EEGsonic#56]
Investigate why the audio crashed at around 9:00 minute into the recording session of the third session.

**Comment by yacineMahdid** [comment]
This happens when we are editing the ranges while the resonance_gui is running.

**Comment by jamiecwebber** [comment]
My suspicion is that we ran out of RAM because of a couple different factors -
1) we had the console open and were doing a lot of console.log(), which is more computationally expensive.
2) some of the files were saved in the .wav format and all of the audio loops are being loaded into memory. Now that we know this is a problem I will use much lighter formats like .ogg
3) at least once it crashed when I hit 'mute', in the Web Audio API there's actually no way to pause and restart a given audio file without having to reload it into memory. To mute the files I used a hack of slowing down the playback rate, but this means it has to resample the audio file and this is really inefficient.

I'm going to mess with it and see if I can reproduce the bug.

**Comment by jamiecwebber** [comment]
...I figured out what was going on. 

While all those ^ would be good changes, it was an issue with one of the audio features. I had tested these on the old ranges from Joe and put a low-pass filter for the audio controlled by one of the PAC signals. It works properly when the PAC is in the range of 0.9-1.1, but the range of the current signal is much wider with Dannie's data and the filter was removing all audio above 30 Hz. If I comment out those lines it sounds great on Dannie's brain signals.

I guess at least now we know what happened.
